sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubClient.cs

```diff
@@ -161,34 +168,35 @@ public EventHubClient(string connectionString) : this(connectionString, null, nu
 
             var sharedAccessSignature = new SharedAccessSignature
             (
-                 BuildAudienceResource(clientOptions.TransportType, eventHubsHostName, eventHubName),
+                 BuildAudienceResource(clientOptions.TransportType, fullyQualifiedNamespace, eventHubName),
                  connectionStringProperties.SharedAccessKeyName,
                  connectionStringProperties.SharedAccessKey
             );
 
             _retryPolicy = new BasicRetryPolicy(clientOptions.RetryOptions);
             ClientOptions = clientOptions;
+            FullyQualifiedNamespace = fullyQualifiedNamespace;
             EventHubName = eventHubName;
-            InnerClient = BuildTransportClient(eventHubsHostName, eventHubName, new SharedAccessSignatureCredential(sharedAccessSignature), clientOptions, _retryPolicy);
+            InnerClient = BuildTransportClient(fullyQualifiedNamespace, eventHubName, new SharedAccessSignatureCredential(sharedAccessSignature), clientOptions, _retryPolicy);
         }
 
         /// <summary>
         ///   Initializes a new instance of the <see cref="EventHubClient"/> class.
         /// </summary>
         ///
-        /// <param name="host">The fully qualified host name for the Event Hubs namespace.  This is likely to be similar to <c>{yournamespace}.servicebus.windows.net</c>.</param>
+        /// <param name="fullyQualifiedNamespace">The fully qualified host name for the Event Hubs namespace.  This is likely to be similar to <c>{yournamespace}.servicebus.windows.net</c>.</param>
```

Comment:
Nit: The fully qualified host name for the Event Hubs namespace -> The fully qualified Event Hubs namespace